core/src/main/scala/org/genivi/sota/core/rvi/SotaServices.scala

```diff
@@ -4,15 +4,15 @@
  */
 package org.genivi.sota.core.rvi
 
+import java.util.UUID
```

Comment:
It was Intellij's auto sort feature. It sorts according to namespace and alphabet. I guess I should probably stop doing this since it just looks confusing in the PRs